### DISCUSSION TITLE
get_labels asking for value not found in Amazon documentation

### DISCUSSION THREAD

**Discussion by godlyatheist** [saleweaver/python-amazon-sp-api#1278] [FAQ]
**Describe the bug**
When calling get_labels, error message "sp_api.base.exceptions.SellingApiBadRequestException: [{'code': 'InvalidInput', 'message': "1 validation error detected: Value null at 'cartonIdList' failed to satisfy constraint: Member must not be null", 'details': ''}]". cartonIdList is not a value associated with the getLabels operation according to documentation. The code below used to work without error.

**To Reproduce**

`from sp_api.base import Marketplaces
from sp_api.api import FulfillmentInbound

credentials = dict(refresh_token='token', lwa_app_id='app_id', lwa_client_secret='secret')

label_blob = FulfillmentInbound(credentials=credentials, marketplace=Marketplaces.US).get_labels(      
    shipment_id = 'FBA0000000',
    PageType = 'PackageLabel_Thermal',
    LabelType = ['UNIQUE','PALLET'] )
print(label_blob['DownloadURL'])
`
**Expected behavior**
No error regarding cartonIdList, returns a URL

**Desktop (please complete the following information):**

Windows 10 build 19045 VSCode 1.85.1

**Additional context**

**Comment by saleweaver** [comment]
I haven't worked with this endpoint for some time, but if I remember correctly, if the code was working, it must have been for a shipment created with something other than a carton feed. The value you need to pass for this kind of shipment should be this

![image](https://github.com/saleweaver/python-amazon-sp-api/assets/50274907/7c849804-7a01-47b9-b72e-9b8672ff24e8)

**Comment by godlyatheist** [comment]
Thank you, I will check my work
